ibtk/include/ibtk/IndexUtilities.h

```diff
@@ -96,6 +96,29 @@ class IndexUtilities
                                                   const double* dx,
                                                   const SAMRAI::hier::Index<NDIM>& ilower,
                                                   const SAMRAI::hier::Index<NDIM>& iupper);
+	
+	/*!
+	 * \brief Convert (i,j,k,d) index for a DOF to an integer mapping on a patch level.
```

Comment:
I'd like these comments to be a bit more clear about what this routine is doing.